lib/repl.js

```diff
@@ -319,9 +320,20 @@ function REPLServer(prompt,
   self.outputStream = output;
 
   self.resetContext();
-  self.bufferedCommand = '';
   self.lines.level = [];
 
+  self.clearBufferedCommand = function clearBufferedCommand() {
```

Comment:
Does this function need to be exposed at all? Would it work to create a function in this file that has the symbol in scope and takes the REPL as an argument?